packages/analytics-js-integrations/src/integrations/Amplitude/browser.js

```diff
@@ -69,9 +72,15 @@ class Amplitude {
     window.amplitude.init(this.apiKey, null, initOptions);
   }
 
-  identify(rudderElement) {
-    logger.debug('in Amplitude identify');
+  isLoaded() {
+    return Boolean(window?.amplitude && window.amplitude?.getDeviceId());
```

Comment:
is there a chance that window does not exist? window?.amplitude checks for it with the "?"
the second check window.amplitude?. is also covered by the first .

Essentially here you just bloat the transpiled code. The original check was sufficient "Boolean(window.amplitude && window.amplitude.getDeviceId())"

I suggest you take this change in as a separate task for all integrations

Reply:
reverted now, will take it in separate pr